README.rst

```diff
@@ -540,7 +540,7 @@ The `projects <https://github.com/microsoft/dowhy/projects>`_ page lists the nex
 Contributing
 ============
 
-This project welcomes contributions and suggestions. For a guide to contributing and a list of all contributors, check out `CONTRIBUTING.md <https://github.com/microsoft/dowhy/blob/main/CONTRIBUTING.md>`_ and our [docs for contributing code](https://github.com/py-why/dowhy/blob/main/docs/source/contributing/contributing-code.rst). Our contributor code of conduct is available [here](https://github.com/py-why/governance/blob/main/CODE-OF-CONDUCT.md). You can also join the DoWhy development channel on Discord: |discord|_
+This project welcomes contributions and suggestions. For a guide to contributing and a list of all contributors, check out `CONTRIBUTING.md <https://github.com/microsoft/dowhy/blob/main/CONTRIBUTING.md>`_ and our `docs for contributing code <https://github.com/py-why/dowhy/blob/main/docs/source/contributing/contributing-code.rst>`. Our contributor code of conduct is available `here <https://github.com/py-why/governance/blob/main/CODE-OF-CONDUCT.md>`. You can also join the DoWhy development channel on Discord: |discord|_
```

Comment:
@emrekiciman There are `_`s missing after the ``>` ``s. rst syntax is a bit of a pain...

Reply:
Those spaces should be there now.